docs/curl.1

```diff
@@ -0,0 +1,2726 @@
+.\" **************************************************************************
```

Comment:
This file was intentionally removed upstream in 4c49b83597969418584344eb0df499d150f8680c.  It should not be re-created with this PR.